doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,242 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-
-        auto result = subscriber->take();
-
-        if(!result.has_error()) 
-        {
-            auto& maybeSample = result.value();
-            if (maybeSample.has_value())
-            {
-                auto& sample = maybeSample.value();
-                void* chunk = sample.get();
-
-                //interpret and process the data
-                const CounterTopic* counter = reinterpret_cast<const CounterTopic*>(chunk);
-            }
-            else
-            {
-                // we received no data
-            }
-        } else {
-            iox::popo::ChunkReceiveResult& error = result.get_error();
-            //handle the error
-        }
-    }
-```
+When multiple publishers have offered the same topic the subscriber will receive the data of all of them (but in
+indeterminate order between different publishers).
 
-### Shutdown
+## Waitset
 
-Once we are done sending, we call ``stopOffer`` at the publisher.
+The easiest way to receive data is to periodically poll whether data is available. This is sufficient for simple use
+cases but inefficient in general, as it often leads to unnecessary latency and wake-ups without data.
 
-```cpp
-publisher.stopOffer();
-```
+The ``Waitset`` can be used to relinquish control (putting the thread to sleep) and wait for user-defined ``events``
+to occur. Usually, these events correspond to the availability of data at specific subscribers. This way we can 
+immediately wake up when data is available and will avoid unnecessary wake-ups if no data is available.
 
-Similarly the subscriber can ``unsubscribe`` to stop receiving any data.
+The Waitset is an implementation of the reactor pattern and is informed with a push strategy that one of the 
+attached events occured at which it informs the user.
 
-```cpp
-subscriber.unsubscribe();
-```
+For more information on how to use the Waitset see 
+[Waitset](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/waitset/README.md).
 
-Both will also be called in the respective destructor if needed (i.e. if the publisher is still offering or the
-subscriber is still subscribed).
+## Listener
+part of #350
 
-### Running an iceoryx system
+## API
 
-Now that we have applications capable of sending and receiving data, we can run the complete iceoryx system.
+The API is offered in two languages, C++ and C. Detailed information can be found in the 
+[C++ example](http://iceoryx.io/getting-started/examples/icedelivery.md) and 
+[C example](http://iceoryx.io/getting-started/examples/icedelivery_in_c.md).
 
-First we need to start RouDi.
+Many parts of the C++ API follow a functional programming approach which is less error-prone. This requires using 
+the monadic types ``cxx::expected`` and ``cxx::optional`` which are introduced 
+[here](http://iceoryx.io/advanced/usage-guide/how-optional-and-error-values-are-returned-in-iceoryx/).
```

Comment:
I would leave it as it is. Repetition is good since the reader will most likely not read it from beginning to the end. They will pick a topic of interest and read only this part. And in this case some redundant links are pretty nice.